openedx/features/colaraz_features/templates/course-access-roles-delete.html

```diff
@@ -32,6 +32,10 @@ <h1>Course Privileges</h1>
         </table>
         <div class="privileges-footer">
             <button type="submit" class="btn cz-btn cz-btn-primary">Delete</button>
+            <a
+                class="btn cz-btn cz-btn-primary"
+                href="${reverse('colaraz_features:course-access-roles-update', args=[','.join(str(obj.id) for obj in objects),])}"
```

Comment:
```suggestion
                href="${reverse('colaraz_features:course-access-roles-update', args=[','.join(str(obj.id) for obj in objects)])}"
```

Reply:
Both represent the same things. ['word'] == ['word',]